### ISSUE TITLE
Bazarr Bad Gateway

### ISSUE THREAD

**Issue by falcomm6** [kelinger/OmniStream#73]
Hi there,

Im having a weird issue with Bazarr only, where it is giving me a bad gateway error. This is _after_ I have entered my basicauth credentials.

I dont have this issue on any other container, and it used to work. I tried deleting the config folder, resetting the yaml to base, reinstalling Traefik. Kinda at a loss.

if anyone has had this and found a fix, pls let me know. I'll be eternally grateful lol

**Comment by kelinger** [comment]
I'm going to have to setup Bazarr in order to replicate this.  However, _usually_ this error indicates that Traefik is sending web hits to the root of your domain (bazarr.mydomain.com) but the site is listening on a subdirectory (bazarr.mydomain.com/bazarr).  This can be changed in the configuration (stop Bazarr, edit the config, and change the setting for where it expects the site to be).  Mind you that none of this is based on looking at Bazarr right now, so it's a guess and may not be exactly how I describe.

After I get some time and get it installed, I'll see what I specifically have to do to make it work.

**Comment by falcomm6** [comment]
Thank you :) I'll see if I can find anything around the subdomain. As said it did work, which is weird.

**Comment by falcomm6** [comment]
Ive tried a few things I found online like - traefik.frontend.rule=PathPrefix:/bazarr but that didn't help, along with updating the config. 

Appreciate your time as always.

**Comment by kelinger** [comment]
I did a clean install of Bazarr (no pre-existing configs) and I did this just by `o+ 307-bazarr.yaml` then `omni up`.  Bazarr was added to Traefik and my DNS and then brought up.  I had no trouble connecting to Bazarr via the web without changing anything.

So, the expectation is that this _should_ work.

I noticed that the configs allow you to set a path.  If you can't get to the web, look for the file "config.ini" located in ~/OmniStream/configs/bazarr/config/ directory.  Three lines down in the General section, I have this:

```
base_url =
path_mappings = []
```
Is that how yours reads?  (Again, this is without changing anything in the YAML for Bazarr, including Traefik labels)

**Comment by falcomm6** [comment]
![image](https://github.com/kelinger/OmniStream/assets/67247329/e7b179f9-e8b7-4bd1-ba5a-4ed3faf2a808)

unfortunately mine reads exactly the same. Hmm.

**Comment by falcomm6** [comment]
As mentioned, it did work, then suddenly stopped working for no knowable reason. it worked for weeks.

**Comment by falcomm6** [comment]
Looks to be an issue with my configs. deleting it now fixed it. no clue where, ill just start again. sorry for some reason this didnt work last time. appreciated.

**Comment by falcomm6** [comment]
Reopening this was ive been able to replicate the fault. When I restart the container (using portainer) after configuring a fresh install, it goes back to bad  gateway. Can you see if this happens to you?

**Comment by kelinger** [comment]
@falcomm6 I reinstalled Bazarr from scratch and it worked (which it did with a vanilla config for you as well).  I was able to get to the interface.  I didn't change anything but I did stop and start the container.  When it was back up, I was still able to connect to the web interface.

I then went through the configuration.  I added a provider (OpenSubtitles.org) and linked it to Sonarr and Radarr (testing the API key with the in-dialog "test" button) and triggered a scan from the tasks.  These are the only configuration changes I made.

The movies fully loaded but I got impatient after about 25% of the TV shows were scanned.  I restarted the Docker container for just Bazarr again and the web interface immediately  said "there's a problem."  But, once Bazarr came up, I hit refresh a few times (initially getting a "page cannot be loaded" error) the Bazarr interface once again presented itself after about 10 seconds.

So, I'm still not able to reproduce this.  Are you changing any other settings besides what I mentioned above?  Are you doing anything different?
